tgui/packages/tgui/interfaces/MassSpec.js

```diff
@@ -1,57 +1,29 @@
 import { round } from 'common/math';
 import { useBackend } from '../backend';
-import { Box, Button, Dimmer, Icon, Section, Slider, Table } from '../components';
+import { Box, Button, Section, Slider, Table } from '../components';
 import { Window } from '../layouts';
 
 export const MassSpec = (props, context) => {
   const { act, data } = useBackend(context);
   const {
-    processing,
     lowerRange,
     upperRange,
     graphUpperRange,
     graphLowerRange,
-    eta,
     beaker1CurrentVolume,
-    beaker2CurrentVolume,
     beaker1MaxVolume,
-    beaker2MaxVolume,
     peakHeight,
     beaker1,
-    beaker2,
     beaker1Contents = [],
-    beaker2Contents = [],
   } = data;
 
   const centerValue = (lowerRange + upperRange) / 2;
 
   return (
     <Window width={490} height={650}>
       <Window.Content scrollable>
-        {!!processing && (
```

Comment:
You're going to have to explain this one to me.

I *don't* want purity removed--that is our current metric for setting two chemicals apart. I want the failed chems and the impure chems removed, but I *do* want to be able to use purity for, say, making pure aiuri heal better than regular aiuri, without having the chemical that blinds you. You get what I mean?

Reply:
Reverted, the spectrometer works as it used to.